batch2/batch/front_end/front_end.py

```diff
@@ -210,7 +210,16 @@
     db = app['db']
 
     batch_id = int(request.match_info['batch_id'])
+
     user = userdata['username']
+    # restrict to what's necessary; in particular, drop the session
+    # which is sensitive
+    userdata = {
+        'username': user,
```

Comment:
It looks like there's still a separate user field in the batches table. Do we also need it here?

Reply:
Yes, because we want to filter on the user for incoming requests and you can't access json inside SQL (at least until we starting using the JSON type).